samples/apps/copilot-chat-app/documentuploadapp/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.CommandLine;
+using System.Text;
+using System.Text.Json;
+using Microsoft.Identity.Client;
+
+namespace SemanticKernel.Service.DocumentUploadApp;
+
+public static class Program
+{
+    public static void Main(string[] args)
+    {
+        var serviceConfig = ServiceConfig.GetServiceConfig();
+        if (!ServiceConfig.Validate(serviceConfig))
+        {
+            Console.WriteLine("Error: Failed to read appsettings.json.");
+            return;
+        }
+
+        var fileOption = new Option<FileInfo>(
+            name: "--file",
+            description: "The file to upload for injection."
+        )
+        { IsRequired = true };
+
+        var userCollectionOption = new Option<bool>(
+            name: "--user-collection",
+            description: "Save the extracted context to an isolated user collection.",
+            getDefaultValue: () => false
+        );
+
+        var rootCommand = new RootCommand(
+            "Sample app that uploads a file to the Document Store."
```

Comment:
How about "This application imports a file to the CopilotChat webapi's document memory store."